src/tests/dfio/SConscript

```diff
@@ -0,0 +1,31 @@
+"""Build dfio plugin"""
+import daos_build
+from prereq_tools import GitRepoRetriever
```

Comment:
(pylint-unused-import) Unused GitRepoRetriever imported from prereq_tools

Reply:
YOu should remove this line